### DISCUSSION TITLE
Using CodeQL action without uploading security alerts

### DISCUSSION THREAD

**Discussion by apple502j** [community/community#32636] [Code Security]
From https://github.com/orgs/community/discussions/29534#discussioncomment-3614341:

> Is it possible to use CodeQL action workflow without uploading security alerts (i.e. as a Pull Request check only)?

A bit of context: I've used LGTM as a way to monitor security issues in a repository as a Pull Request check. However I don't want them to clutter Code Scanning section of Security tab. Is there a way to disable that behavior?

**Comment by edoardopirovano** [comment]
Greetings, thanks for getting in touch with this question! It is indeed possible to set `upload: false` in the `with:` block of the `github/codeql-action/analyze` step in your workflow to disable uploading of the results to Code Scanning. This will prevent alerts from getting to the Security tab, although it will also stop comments from being posted on your Pull Request with the relevant alerts (since this also happens in the Code Scanning backend) which is an unfortunate side effect. The check itself will still fail if there's new alerts, though, so if that's the only behaviour you want then this should work for you.

**Reply by apple502j** [reply]
Ah, makes sense. I could probably make a new action that uses checks API instead of security API. Thanks for answering!
